packages/blog/index.ts

```
@@ -156,13 +156,13 @@ class BlogDetailHandler extends BlogHandler {
     }
 
     @param('did', Types.ObjectId)
-    async postStar(domainId: string, did: ObjectId) {
+    async postStar({ }, did: ObjectId) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)